opm/parser/eclipse/EclipseState/Schedule/Group/GConSump.hpp

```diff
@@ -0,0 +1,51 @@
+/*
+  Copyright 2019 Equinor ASA.
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef GCONSUMP_H
+#define GCONSUMP_H
+
+#include <map>
+#include <string>
+
+#include <opm/parser/eclipse/Deck/UDAValue.hpp>
+
+namespace Opm {
+
+    class GConSump {
+    public:
+        GConSump();
```

Comment:
When you just want the absolute default default constructor you can do:
```
GConSump() = default;
```
and do not need to implement anything in the cpp file.